modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -118,7 +118,17 @@ v = f_y \times y'' + c_y
 tangential distortion coefficients. \f$s_1\f$, \f$s_2\f$, \f$s_3\f$, and \f$s_4\f$, are the thin prism distortion
 coefficients. Higher-order coefficients are not considered in OpenCV.
 
-The next figures show two common types of radial distortion: barrel distortion (typically \f$ k_1 < 0 \f$) and pincushion distortion (typically \f$ k_1 > 0 \f$).
+The next figures show two common types of radial distortion: barrel distortion
+(1 + k_1 r^2 + k_2 r^4 + k_3 r^6 monotonically decreasing)
+and pincushion distortion (1 + k_1 r^2 + k_2 r^4 + k_3 r^6 monotonically increasing).
+ Radial distortion is always monotonic for real lenses,
```

Comment:
Pedantic: left align the text to be consistent with the surrounding text.
Maybe too many break lines.